tony-core/src/main/java/com/linkedin/tony/util/ParserUtils.java

```diff
@@ -202,15 +203,15 @@ public static JobMetadata parseMetadata(FileSystem fs, Path jobFolderPath, Strin
   }
 
   public static List<JobEvent> mapEventToJobEvent(List<Event> events) {
-    List<JobEvent> jobEvents = new ArrayList<>();
-    for (Event e : events) {
-      JobEvent wrapper = new JobEvent();
-      wrapper.setType(e.getType());
-      wrapper.setEvent(e.getEvent());
-      wrapper.setTimestamp(e.getTimestamp());
-      jobEvents.add(wrapper);
-    }
-    return jobEvents;
+    return events.stream().map(ParserUtils::initJobEvent).collect(Collectors.toList());
+  }
+
+  private static JobEvent initJobEvent(Event e) {
```

Comment:
this looks weird. Look like it should be a method in JobEvent class?

Reply:
Ahh I see. That makes more sense actually. Will update soon.